UI/obs-app.cpp

```diff
@@ -766,7 +732,9 @@ bool OBSApp::MigrateLegacySettings(const uint32_t lastVersion)
 		hasChanges = true;
 	}
 
-	return hasChanges;
+	if (hasChanges) {
+		userConfig.SaveSafe("tmp");
+	}
```

Comment:
Leftover debug code? Or what is the purpose of this?

Reply:
It's necessary to actually persist the migration changes to `basic.ini`.

Reply:
I thought this was the backup extension but apparently we allow the caller to specify the tmp extension too, never mind.